### ISSUE TITLE
[REQ] Back and forward navigation

### ISSUE THREAD

**Issue by gmenzel** [glushchenko/fsnotes#335]
It would be great to be able to go back to the note that I was viewing before. When following a few links it's easy to lose track of where you were.

**Comment by peavine** [comment]
This is an excellent idea. I'm constantly switching from note to note in two or more folders, and this would make that so much easier. I'd especially like to have this as a menu item in the top menu bar, as this would allow me to assign shortcut keys of my choosing.

**Comment by gingerbeardman** [comment]
+1 for this @glushchenko

**Comment by glushchenko** [comment]
Added in 4.1.3

**Comment by peavine** [comment]
I just installed FSNotes 4.1.3 and tried out the "Prev Note" and "Next Note" menu items and they work great. I also tried the keyboard shortcuts and they also worked fine.

I tried to assign different keyboard shortcuts for these items using System Preferences, but the new keyboard shortcuts do not work and are not shown in the FSNotes menu for these items.  I tried many different keyboard shortcuts, thinking there might be some conflict, but none of them worked.

Am I doing something worng? I have reassinged many other FSNotes keyboard shortcuts (e.g. move and print) and this worked as expected.
